CMakeLists.txt

```diff
@@ -5,6 +5,18 @@ set(CMAKE_CXX_EXTENSIONS off)
 set(CMAKE_CXX_STANDARD 23)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+find_package(glslang CONFIG)
+find_package(nlohmann_json CONFIG)
+find_package(spirv_cross_core CONFIG)
```

Comment:
We need to verify that version of spirv-cross can parse SPIRV 1.6. Latest release of spirv-cross was in 2021 and it cannot parse newer extensions. We cannot determine which patches are applied by distributive maintainers by comparing versions because all will have same 2021 version

Reply:
There is [many new tags](https://github.com/KhronosGroup/SPIRV-Cross/tags) that can be used. We can also create a `FindSPIRV-Cross.cmake` module that checks anything: function present in header, symbol present in shared library, or even a small program that try to parse a SPIRV 1.6 sample.

Reply:
You can try if you want :)

Reply:
Libraries not compatible with SPIRV 1.6 are now rejected ;)